cleaners/epiphany.xml

```diff
@@ -30,13 +30,16 @@
     <action command="delete" search="file" path="~/.gnome2/epiphany/ephy-favicon-cache.xml"/>
     <action command="delete" search="walk.files" path="~/.gnome2/epiphany/favicon_cache/"/>
     <action command="delete" search="walk.files" path="~/.gnome2/epiphany/mozilla/epiphany/Cache/"/>
+    <action command="delete" search="file" path="~/.cache/epiphany-browser/icondatabase/WebpageIcons.db"/>
+    <action command="delete" search="walk.files" path="~/.cache/epiphany-browser/"/>
```

Comment:
The first action `~/.cache/epiphany-browser/icondatabase/WebpageIcons.db` is redundant to to the next action `~/.cache/epiphany-browser/`